task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -423,12 +427,16 @@ def _close_unused_sockets(*sockets):
         for sock in sockets:
             sock.close()
 
-    def _on_child_started(self, ti: TaskInstance, path: str | os.PathLike[str], requests_fd: int):
+    def _on_child_started(self, *, requests_fd: int, **kwargs):
         """Send startup message to the subprocess."""
+        ti: TaskInstance = kwargs["ti"]
+        path: str | os.PathLike[str] = kwargs["path"]
```

Comment:
Do we need to do this change? Other than mypy thinking that subclasses have to have the same signature, isn't this functionally identical?

Reply:
it's just for mypy, and whatever god enforces liskov substitution principle